php/Terminus/Models/Upstream.php

```diff
@@ -37,7 +45,7 @@ public function fetch(array $args = [])
     {
         $options = array_merge(['options' => ['method' => 'get',],], $this->args, $args);
         $results = $this->request->request($this->url, $options);
-        $this->attributes = $this->parseAttributes($results['data']->upstream);
+        $this->attributes = (object)array_merge((array)$this->attributes, (array)$this->parseAttributes($results['data']->upstream));
```

Comment:
@TeslaDethray could you review my previous two questions here? I'd love to merge this today!